kythe/javatests/com/google/devtools/kythe/platform/kzip/KZipReaderTest.java

```diff
@@ -54,6 +54,24 @@ public void testOpenFailsForMissingFile() throws KZipException {
     }
   }
 
+  @Test
```

Comment:
Is there a test for a file that is valid and contains both proto and json?

Reply:
That's covered in KZipWriterTest::testWriteAll